matsim/src/main/java/org/matsim/core/router/MainModeIdentifierImpl.java

```diff
@@ -42,6 +42,9 @@ public String identifyMainMode( final List<? extends PlanElement> tripElements)
 			// Marcel's SBB raptor returns access/egress_walk to and from pt, and transit_walk only
 			// for direct walk (and presumably in between pt legs, if necessary).  kai/gregor, sep'18
 		}
+		if ( mode.endsWith( "_fallback" ) )  {
+			return mode.split( "_fallback" )[0] ;
```

Comment:
I would probably use `mode.substring(0, mode.length() - 9)` instead of `split`, as `split` is (a) a more expensive method, and (b) while it may works it looks counter-intuitive to me (we're not splitting, we want to cut off a part).

Reply:
perfectly fine with me.  done